.github/workflows/main.yml

```diff
@@ -1,6 +1,10 @@
 name: Build & Test
 
-on: [push]
+on:
+  push:
+    branches: main
+  pull_request:
+    branches: '**'
```

Comment:
We can remove this line if we want to run this workflow on every PRs
```suggestion
```

Reply:
That's what I did at first but a value-less field/property looked odd visually to me so I've added the value~ here. This should also execute for all branches

I don't mind adjusting it to the proposed version if that's what you prefer